### DISCUSSION TITLE
Could this be a box64 issue

### DISCUSSION THREAD

**Discussion by RZMOVEMENT** [ptitSeb/box64#2146] [Q&A]
Hey sorry if its not the place but i wanted to rule out options,Tried to run this game but no emulator on Android can do it... i get the same error on street fight vi demo, but ME:LE doesn't have deunvo and won't run even with steam/ea app 
![Screenshot_2024-12-12-15-34-27-245_com winlator](https://github.com/user-attachments/assets/cfa33924-31db-4092-ae36-edf08715a392)
![Screenshot_2024-12-12-15-34-04-077_com micewine emu](https://github.com/user-attachments/assets/d421e4ae-cedd-4821-8b47-ce0a845c57bf)
![Screenshot_2024-11-29-01-10-41-252_com winlator](https://github.com/user-attachments/assets/c3af609f-0b38-42ae-adaa-754adbd7496a)

The screenshot that includes steams log is sfvi

**Comment by RZMOVEMENT** [comment]
Another log I forgot to upload 
![Screenshot_2024-12-12-15-59-04-290_com micewine emu](https://github.com/user-attachments/assets/cdbd1ab6-fe48-4fcb-8400-badfd9ec6a5f)

**Comment by ptitSeb** [comment]
This seems to be coming from MiceWine. You should ask MiceWine devs, not me.

**Reply by RZMOVEMENT** [reply]
> This seems to be coming from MiceWine. You should ask MiceWine devs, not me.

Ok but I've uploaded logs from winlator as well just pointing out, so it aint a box issue u are saying

**Comment by ptitSeb** [comment]
Well, I don't know because it doesn't looks like the same errors to me. MiceWine is still unstable and can crash because Signal handling is not 100% correct for now.
The message from winelator look slike missing libs, so not an box64 error directly.

Also, what is ME:LE, is it Mass Effect: Legendary Edition?

Also also; those are not Logs. It's screenshot of a very small portion of a log, and is basically useless here.

**Reply by RZMOVEMENT** [reply]
> Well, I don't know because it doesn't looks like the same errors to me. MiceWine is still unstable and can crash because Signal handling is not 100% correct for now. The message from winelator look slike missing libs, so not an box64 error directly.
> 
> Also, what is ME:LE, is it Mass Effect: Legendary Edition?
> 
> Also also; those are not Logs. It's screenshot of a very small portion of a log, and is basically useless here.

Thanks for the answers, i know it ain't the full just wanted to mainly show the error message that preventing the game to even boot
 me le =mass effect legendary edition 
and i showed micewine as another example that the game won't boot(just like all the other emulators : winlator,mobox,darkos etc..)

**Reply by ptitSeb** [reply]
If ME:LE is the remake regroup the 3 mass effect in one game on steam, then, last time I tried it, it was working but they needed safeflags=2 and strongmem=1 to work (else there was random freeze). But that was starting the game from Linux/Steam.
